etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/ruleengine/transformation/custom/convertToOmlOrder.java

```diff
@@ -0,0 +1,24 @@
+package gov.hhs.cdc.trustedintermediary.etor.ruleengine.transformation.custom;
+
+import gov.hhs.cdc.trustedintermediary.context.ApplicationContext;
+import gov.hhs.cdc.trustedintermediary.etor.metadata.EtorMetadataStep;
+import gov.hhs.cdc.trustedintermediary.etor.ruleengine.FhirResource;
+import gov.hhs.cdc.trustedintermediary.etor.ruleengine.transformation.CustomFhirTransformation;
+import gov.hhs.cdc.trustedintermediary.external.hapi.HapiOrderConverterHelper;
+import gov.hhs.cdc.trustedintermediary.wrappers.MetricMetadata;
+import java.util.Map;
+import org.hl7.fhir.r4.model.Bundle;
+
+/** Custom transformation to convert an ORM order to OML. */
+public class convertToOmlOrder implements CustomFhirTransformation {
+
+    private final MetricMetadata metadata =
+            ApplicationContext.getImplementation(MetricMetadata.class);
+
+    @Override
+    public void transform(FhirResource<?> resource, Map<String, String> args) {
+        Bundle bundle = (Bundle) resource.getUnderlyingResource();
+        HapiOrderConverterHelper.convertToOmlOrder(bundle);
```

Comment:
I was thinking that the logic was going to be part of the transformation object.  But instead, it calls a the helper class that has the logic there. What would be the difference if we have the logic in here, so if we need to change  the logic, we go to`convertToOmlOrder.java` instead of the helper class, and the helper class can be a wrapper to that calls all the transformation objects?

Reply:
We could have the logic here. The idea of having the helper is to have functions that could be used by multiple transformations. I don't think we have that right now but I'd propose to leave this refactoring to a separate PR